TaskService/Startup.cs

```diff
@@ -11,6 +9,8 @@ public partial class Startup
         public void Configuration(IAppBuilder app)
         {
             ConfigureAuth(app);
+
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
This shouldn't be needed if we use the latest .NET framework versions, @aremo-ms 
Did you check that it is needed?

Reply:
In general I couldn't reproduce that issue, so I've added it.

Reply:
@jmprieur 
As I can see the value before setting, it equals "SSL3 | Tls", so I think we do need to force it. Tell me know what you think.
![image](https://user-images.githubusercontent.com/85962119/139735248-3bfb83be-e643-4502-9f99-86d3beafec52.png)

Reply:
We are progressing @aremo-ms 
Now the question is why isn't the web app proposing TLS 1.3 too? it's the same framework version? Maybe TLS1.2 is set

Reply:
this is exactly same version. As I said, I couldn't reproduce the original issue on my local machine, so I don't know when it doesn't work.
Inside Client's ConfigureAuth() we set it to 1.2 as part of [original code](https://github.com/Azure-Samples/active-directory-b2c-dotnet-webapp-and-webapi/blob/196df0ee441791cb57e887c0427a9c20d9b78a49/TaskWebApp/App_Start/Startup.Auth.cs#L29). I can add same setting to Server ConfigureAuth()

Reply:
Ok. So I was suggesting to no longer set it in the Client ConfigureAuth() (and no longer in the service one). This should fix it given the versions of .NET fw used.

Reply:
ok, done